lua/lazyvim/plugins/extras/lang/toml.lua

```diff
@@ -0,0 +1,34 @@
+return {
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "toml" })
+      end
+    end,
+  },
+  {
+    "williamboman/mason.nvim",
+    opts = function(_, opts)
+      opts.ensure_installed = opts.ensure_installed or {}
+      vim.list_extend(opts.ensure_installed, { "taplo" })
+    end,
+  },
+  {
+    "stevearc/conform.nvim",
+    opts = {
+      formatters_by_ft = {
+        toml = { "taplo" },
+      },
+    },
+  },
+  {
+    "neovim/nvim-lspconfig",
+    ft = { "toml" },
```

Comment:
Why add `ft` to `nvim-lspconfig`? The LSP servers already have filetypes and patterns to search for to be loaded upon. Also, since the Mason spec only installs `taplo` LSP server, you can remove it as a whole, since every LSP server that gets defined in `nvim-lspconfig` will get automatically installed. Of course, this is just a suggestion from my part for the Mason spec and you're free to keep it around for reasons of clarity (but it doesn't really do anything).

Reply:
hi @dpetka2001 , apologies. I forgot to remove that. I use ft everywhere in my own configs.

Reply:
I removed ft and force pushed the changes. Regarding Mason, I like to have those for clarity in my configs.

Reply:
Please no need to apologize for anything. I'm just a regular user like you. I really don't feel comfortable. I just pointed out something that isn't needed and just a suggestion for Mason.

Reply:
You might want to do the same for your `elm` PR, since you already made the change here.